spec/author_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../classes/author'
+
+# Describe the Author class and its behavior
+describe Author do
+  let(:author) { Author.new('Stephen King') }
+
+  it 'has a name attribute' do
+    expect(author.name).to eq('Stephen King')
+  end
+
+  it "can add an item to the author's collection" do
+    # Create a sample item
+    item = double('Item')
+
+    expect(item).to receive(:set_author).with(author)
+
```

Comment:
I think you can add a test that checks that the **item** is added to the **items** array when the add_item method is called.